src/main/java/de/doubleslash/keeptime/common/ColorTimeLine.java

```diff
@@ -0,0 +1,42 @@
+package de.doubleslash.keeptime.common;
+
+import java.time.Duration;
+
+import de.doubleslash.keeptime.controller.Controller;
+import de.doubleslash.keeptime.model.Model;
+import de.doubleslash.keeptime.model.Work;
+import javafx.scene.canvas.Canvas;
+import javafx.scene.canvas.GraphicsContext;
+import javafx.scene.paint.Color;
+
+public class ColorTimeLine {
+
+   private final Canvas canvas;
+   private final Model model;
+   private final Controller controller;
+
+   public ColorTimeLine(final Canvas canvas, final Model model, final Controller controller) {
+      this.canvas = canvas;
```

Comment:
we should not have a dependency to the model or the controller. we should be able to tell it the needed things to draw. the class should not have to calculate this on its own.

Reply:
DONE! noice :)